src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19500.cs

```diff
@@ -19,6 +19,11 @@ public void TextInEditorShouldScroll()
 			_ = App.WaitForElement("editor");
 			App.ScrollDown("editor");
 
+#if MACCATALYST // In Catalyst scroll down is not effective so here we retry one more time to address the flakyness.
+			Thread.Sleep(500);
+			App.ScrollDown("editor");
```

Comment:
Modifying the `swipePercentage` parameter helps to fix the problem?

Reply:
@jsuarezruiz, This test failure is not due to the swipe area percentage. In the failure scenario, the swipe action did not initiate at all. Therefore, we have implemented a retry mechanism instead of increasing the swipe percentage. Please let me know if you still have any concerns regarding this approach.